lib/linguist/heuristics.rb

```diff
@@ -380,6 +380,24 @@ def call(data)
       end
     end
 
+    disambiguate ".pod" do |data|
+      if /^[\s&&[^\n]]*=\w+$/.match(data)
+        if /^[\s&&[^\n]]+=(begin|cut|pod)/.match(data)
```

Comment:
Perl 6 pod does not use =cut or =pod at all so those should be removed from this match. In addition, all =begin blocks in Perl 6 require something coming after it, so this seems redundant with the check below?

Reply:
Maybe it could be changed to (begin|head) so it recognizes more common indented directives?